docs/modules/ROOT/pages/documentation/payara-server/server-configuration/http/protocols.adoc

```diff
@@ -13,3 +13,604 @@ commands. The protocol configurations can be found in the admin console under
+
+
+[[configuration-ssl-client-auth]]
+=== Client Authentication
+
+When enabled, clients will be required to authenticate themselves to the server.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.client-auth-enabled=[true/false]`
+
+[[configuration-ssl-cert-nickname]]
+=== Certificate Nickname
+
+The alias of the certificate to be supplied on secure requests. The certificate should be present in the server keystore.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.cert-nickname=value`
+
+[[configuration-ssl-keystore]]
+=== Key Store
+
+The name of the keystore file. A keystore stores the certificate to be sent by the server with responses.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.key-store=value`
+
+[[configuration-ssl-truststore]]
+=== Trust Store
+
+The name of the trust store file. The trust store stores trusted certificates.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.trust-store=value`
+
+[[configuration-ssl-trust-algorithm]]
+=== Trust Algorithm
+
+The name of the trust management algorithm (e.g. `PKIX`) to use for certification path validation.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.trust-algorithm=value`
+
+[[configuration-ssl-max-cert-length]]
+=== Max Certificate Length
+
+Maximum number of non self-issued intermediate certificates that can exist in a certification path.
+This is only valid if the trust algorithm is `PKIX`.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.trust-max-cert-length=value`
+
+[[configuration-ssl-enabled-ciphers]]
+=== Enabled Ciphers
+
+A list of the server enabled ciphers. This string is stores as a comma separated list of the enabled ciphers
+with a `+` or a `-` at the start of each, depending on whether the cipher is enabled or disabled.
+E.g. `+TLS_RSA_WITH_AES_256_CBC_SHA,+SSL_RSA_WITH_3DES_EDE_CBC_SHA`.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.ssl3-tls-ciphers=value`
+
+[[configuration-ssl-handshake-timeout]]
+=== Handshake Timeout
+
+The timeout, in millis, for a handshake. After this timeout the handshake will be aborted.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.handshake-timeout-millis=value`
+
+[[configuration-ssl-tls-rollback]]
+=== TLS Rollback
+
+Whether TLS rollback is enabled or not.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.ssl.tls-rollback-enabled=[true/false]`
+
+== Protocols: Asadmin Commands
```

Comment:
The title is a bit confusing. Maybe rename it to just as "Main Asadmin commands" instead?